tasker/core/task_executor_main.py

```
@@ -917,7 +917,35 @@ def cleanup(self):
+                            cleanup_errors.append(f"Task {task_id} stdout temp file cleanup failed: {e}")
+
+                    # Clean up stderr temp file if exists
+                    stderr_file = result.get('stderr_file')
+                    if stderr_file:
+                        try:
+                            import os
+                            if os.path.exists(stderr_file):
+                                os.unlink(stderr_file)
+                        except (OSError, IOError) as e:
+                            cleanup_errors.append(f"Task {task_id} stderr temp file cleanup failed: {e}")
+        except Exception as temp_cleanup_error:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)